assets/stories/skate-components/icons/emptyDetourTableIcon.stories.tsx

```diff
@@ -0,0 +1,15 @@
+import type { Meta, StoryObj } from "@storybook/react"
+import { EmptyDetourTableIcon } from "../../../src/components/detoursTable"
+
+const meta = {
+  component: EmptyDetourTableIcon,
+  parameters: {
+    layout: "centered",
+    stretch: false,
```

Comment:
question: `stretch: false` is the default behavior, does this need to be specified?

Reply:
I copied that from the [detour end icon story](https://github.com/mbta/skate/blob/cb2763e6ed74ba194033bc6487e2b68e3b3a7e9f/assets/stories/skate-components/detours/markers/startIcon.stories.tsx#L9), which I think I copied from the [map location search marker story](https://github.com/mbta/skate/blob/cb2763e6ed74ba194033bc6487e2b68e3b3a7e9f/assets/stories/skate-components/map/markers/locationSearchMarker.stories.tsx).

But it does look like removing that property doesn't change anything. Perhaps I'll remove it from some of the other stories as well (as a follow-up).

Reply:
Oh and I'm the one who added it to the map location search marker story too, so I have no idea what's going on lol.